From 35a51a98259e8285ca0b79a29f3bd7633320cfd8 Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Fri, 15 May 2015 22:53:22 +0800
Subject: [PATCH] MLK-10900 ARM: dts: imx: update 996MHz setpoint voltage on
 i.mx6sx

Update the i.MX6SX operating points to comply with the latestd datasheet.
According to the latest i.MX6SX datasheet of Rev.0, 2/2015, the setpoint
voltage of 996MHz is 1.25V. For the general setting, add a 25mV margin to
cover the board IR drop.

Signed-off-by: Bai Ping <b51503@freescale.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 72254ea..67120ac 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -222,14 +222,14 @@
 &cpu0 {
 	operating-points = <
 		/* kHz    uV */
-		996000  1250000
+		996000  1275000
 		792000  1175000
 		396000  1175000
 		198000	1175000
 		>;
 	fsl,soc-operating-points = <
 		/* ARM kHz      SOC uV */
-		996000	1250000
+		996000	1275000
 		792000	1175000
 		396000	1175000
 		198000	1175000
-- 
1.8.0

